lib/std/crypto/aes.zig

```diff
@@ -0,0 +1,606 @@
+// Based on Go stdlib implementation
+
+const std = @import("../std.zig");
+const mem = std.mem;
+const testing = std.testing;
+
+// Apply sbox0 to each byte in w.
+fn subw(w: u32) u32 {
+    return u32(sbox0[w >> 24]) << 24 | u32(sbox0[w >> 16 & 0xff]) << 16 | u32(sbox0[w >> 8 & 0xff]) << 8 | u32(sbox0[w & 0xff]);
+}
+
+fn rotw(w: u32) u32 {
+    return w << 8 | w >> 24;
+}
+
+// TODO: encrypt and decrypt could probably be unified
```

Comment:
I think this is clear enough from someone who is seriously interested in working on the implementation. No need to create a chore for our future selves to clean up this TODO. (See #363)

Reply:
thanks -- these were really more like questions I wanted to ask of reviewers. In the future I'll put them in the PR body rather than in the source.